realm/realm-library/src/main/java/io/realm/RealmObject.java

```diff
@@ -354,6 +365,30 @@ public final boolean load() {
     }
 
     /**
+     * Adds a change listener to a RealmObject.
```

Comment:
Really, I thought it did that as well at least for the global ChangeListener, but if not, then yes, that wording is fine 👍